cli/setup.cfg

```diff
@@ -0,0 +1,46 @@
+# This file is used to configure your project.
+# Read more about the various options under:
+# http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
+
+[metadata]
+name = ripsaw-cli
+description = To run benchmarks against openshift/kubernetes from the CLI
+author = red-hat-performance
+author_email = perf-dept@redhat.com
+license = Apache License 2.0
+long_description = file: README.md
+long_description_content-type = text/markdown; charset=UTF-8
+classifiers =
+    Development Status :: 4 - Beta
+    Programming Language :: Python
+version = file: version.txt
+home-page = https://github.com/cloud-bulldozer/benchmark-operator/cli
+
+[options]
+zip_safe = False
+packages = find:
+include_package_data = True
+# Add here dependencies of your project (semicolon/line-separated), e.g.
+install_requires =
+    python-benedict==0.24.0
+    click==8.0.1
+    kubernetes==17.17.0
+    urllib3==1.25.7
+    PyYAML==5.4.1
+    elasticsearch==7.13.4
+    GitPython==3.1.18
+python_requires = >=3.8
+
+[options.extras_require]
+tests = pytest; pytest-cov; tox; pytest-kind; ElasticMock; pytest-env
```

Comment:
Can we add these on separate lines to match the format from above?

Reply:
fixed